performance-storage-service/pss_project/api/serializers/rest/metrics/MemoryMetricsSerializer.py

```diff
@@ -3,15 +3,15 @@
 
 
 class MemoryItemSummarySerializer(Serializer):
-    avg = DecimalField(max_digits=10, decimal_places=4, coerce_to_string=False)
+    avg = DecimalField(decimal_places=4, coerce_to_string=False)
```

Comment:
Oh man I thought max_digits was a required field. Thats awesome that you can leave it out. I probably should do that in more places

Reply:
no i cannot... i added it back